gradle/libs.versions.toml

```diff
@@ -7,6 +7,7 @@ json = "20240303"
 kotlin = "1.9.0"
 gms = "4.4.2"
 ktfmt = "0.17.0"
+sonar = "4.4.1.3373"
```

Comment:
This was a good initiative, but you should have done the fix in a separate PR. This commit does not belong here.